src/gui/render/graphics_shaders/quad_renderer_multisample.hpp

```diff
@@ -58,10 +58,10 @@ class QuadRendererMultisample : public QuadRenderer {
      */
     explicit QuadRendererMultisample(ShaderHandler shader_handler = ShaderHandler());
 
-    inline ~QuadRendererMultisample() {
+    /*inline ~QuadRendererMultisample() {
         glDeleteBuffers(1, &quad_vertexbuffer_);
         glDeleteProgram(program_id_);
-    }
+    }*/
```

Comment:
uhh is this right

Reply:
not sure. I was just trying things. It is the same as the base class so I though it was redundant.

Reply:
ok then. delete it